gnuplot_files/plot_memory_base.gp

```diff
@@ -20,8 +20,8 @@ set output "__OUT_FILENAME__"
 set style fill solid border -1
 set grid xtics
 set grid ytics
-set xtics 100000
-set yrange [0:100]
+set xtics 1000000
```

Comment:
Ok I'd say remove it then -- since without it, these scripts are easier to use in the general case (where we're using them to eyeball things), and any time we're using these scripts to show data in the paper, we'll do a bunch of re-formatting anyway.